raiden/blockchain_events_handler.py

```diff
@@ -52,27 +53,30 @@ def handle_tokennetwork_new(raiden: "RaidenService", event: Event):
     token_network_address = args["token_network_address"]
     token_address = typing.TokenAddress(args["token_address"])
     block_hash = data["block_hash"]
+    # from raiden.network.rpc.client import check_address_has_code
+    # check_address_has_code(raiden.chain.client, token_network_address, 'TokenNetwork')
+    try:
```

Comment:
I dont understand the changes. 

The method that raises the `AddressWithoutCode` is `check_address_has_code` right? 

And that code is commented and is not inside the `try-catch `block

Reply:
The function check_address_has_code is called when trying to get the token_network_proxy as a check... if it raises the AddressWithoutCode exception, the try-catch block will handle it, and not break the node

Reply:
I'll delete the comments, was just for debugging

Reply:
Ok got it !